### PR TITLE
WIP: fix static member encoding

### LINKED ISSUE #425: Unable to persist @nearBindgen classes with static properties
**Describe the bug**

Compiler error when attempting to persist a `@nearBindgen` class that has a `static` property.  Can't say for sure that this is a bug as I'm not sure what the expected behavior should be -- either ignoring static properties or storing seperately.

**To Reproduce**

Attempt to persist, via `persist(this)` or `storage.set("KEY", this)`, a @nearBindgen'd class that has a static property.
```ts
import { persist } from 'near-sdk-as';

@nearBindgen
class Contract {
  static account: string = "contract.near"; 
}

export function init(): void {
  let c = new Contract();
  persist(c);
}
```

**Expected behavior**

Unclear what expected behavior should be.

**Screenshots**
```sh


> greeter@0.0.1 build
> asb

ERROR TS2339: Property 'account' does not exist on type 'assembly/index/BaseContract'.

     this.account = obj.has("account") ? decode<string, JSON.Obj>(obj, "account"): "contract.near";
          ~~~~~~~
 in assembly/index.ts(25,10)

ERROR TS2339: Property 'account' does not exist on type 'assembly/index/BaseContract'.

     encode<string, JSONEncoder>(this.account, "account", encoder);
                                      ~~~~~~~
 in assembly/index.ts(32,38)
```

**Desktop (please complete the following information):**
 - OS: Mac OS 11.1, Arm64
 - Version [e.g. 22]: 2.2.0

### PR DESCRIPTION
**Pull request by mehtaphysical**
Fixes #425 and #46

No longer encode/decode static members.

### CODE REVIEW

**Comment by mehtaphysical** [comment]
Still need to add tests

**Comment by willemneal** [comment]
Great!

File: simulator/__tests__/singleton.spec.ts [hunk 1]
```diff
@@ -16,34 +15,42 @@ function getErrorMsg(res: any) {
...
+      );
     }
-  })
-})
-
+  });
+});
 
-describe("cross contract calls", () => {
+describe("Singleton Contract", () => {
```
Comment (willemneal) [inline simulator/__tests__/singleton.spec.ts:53]: Nice catch

**Review by willemneal — APPROVED** [review]
LGTM!

**Pull request merged**